### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by gearmesh** [evs-chris/Hearty-McWeatherface#2] [Ideas]
Consider putting "01-Main.png" image of the front page of this project. This looks like an awesome clock face. 
While I am  learning and making my own clock face which is based on another project, it does not compare to your work here, which 
is spectacular.

**Comment by evs-chris** [comment]
I'm slowly getting docs together for this to list on the fitbit gallery, which doesn't really offer enough space to describe everything. I'm aiming to have full breakdowns of each view at some point, but as you may be able to tell, design/graphics isn't exactly my forte, though I did learn a good deal more about manipulating svgs in this project.

**Comment by gearmesh** [comment]
I was doing some FitBit watch face training today and that filled a lot of
gaps of information. Whilethe lesson was poorly written near the end. I
still learned a lot.
I need to do some reading up on placement of text and pngs on the display.
I like your graphics layout. Sure, it is tight to fit on the display,
but your face has way more information than most. It's not some stupid
scene with time on it.
Greg
